packages/terra-tag/src/Tag.scss

```diff
@@ -0,0 +1,76 @@
+
+:local {
+  .tag {
+    border-radius: 0.2142rem;
```

Comment:
I don't think we should need to make it themeable, but lets check with @neilpfeiffer on whether it should be.

Reply:
Actually, looking at your callouts, it looks like this doesn't need to be themeable.